pytests/tests/test_clean.py

```diff
@@ -19,28 +19,28 @@ def teardown_test(utils):
 
 def test_clean_no_args(utils):
     ret = utils.run([ 'tdnf', 'clean' ])
-    assert (ret['retval'] == 135)
+    assert (ret['retval'] == 903)
```

Comment:
do we have any hash defined for these ret values? it is returning from tdnf, can we use readable names.

Reply:
Thought about this, felt it was not needed. Keerthana brought this up earlier. I will explain in a regular comment so no one misses it.